lib/sirius_api/base_authorizer.rb

```diff
@@ -0,0 +1,81 @@
+require 'sirius_api/errors'
+
+module SiriusApi
+  Permission = Struct.new(:http_method, :url, :options) do
+    def ==(other)
+      http_method == other.http_method && url == other.url
+    end
+  end
+
+  class BaseAuthorizer
```

Comment:
This is very important class, so please add yardoc to all public methods and brief example how to use this DSL.